tests/jsx_last_line/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -48,3 +48,48 @@ exports[`last_line.js 2`] = `
 </SomeHighlyConfiguredComponent>;
 
 `;
+
+exports[`single_prop_multiline_string.js 1`] = `
+<path d="M4.765 16.829l3.069-2.946 5.813 5.748
+  11.33-11.232 3.006 3.18-14.36 14.080z" />;
+
+<Component text="Text
+  with
+  newlines">Content</Component>;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<path
+  d="M4.765 16.829l3.069-2.946 5.813 5.748
+  11.33-11.232 3.006 3.18-14.36 14.080z"
+/>;
```

Comment:
Shouldn’t this end up on the same line since `jsxBracketSameLine` is `true`?

Reply:
Nope

**Prettier 1.10.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAwgHSgAhwBwCcJ8BBAXkxABtoBzWqB+x5p1jl+q7PIkgEKUaXdqLYTOYpjygB6AHwBuEABoQJGAEtoAZ2SgAhoWIB3AArGE+lIYBuELQBM1IQ7pjIYhAK5x1AEaEhmAA1nAwAMr4IVpMXr7+IE4QYMgAZobUuklxOYQw5sF0ALaGGVk56gBWugAeAsFhEZGGJXAAMnFwCX7qED4w+IMATBXZSTGE+cggAYYBAJ6MrkRxMADqzjAAFsgAHAAM6vw5G8H4s0Rw+XY96oRwAI4+Wo9FhqXlSJkT6jklLS9JK6OIMOAARR8EHg4yqIBgCy2Tl2yBG6m8hi01DB6AgJTKsyg0HuIB8OQAKgsbL8cgBfOlAA)
```sh
--jsx-bracket-same-line
```

**Input:**
```jsx
<C
  propA="longlonglonglonglonglonglonglonglong"
  propB="longlonglonglonglonglonglonglonglong"
/>;
```

**Output:**
```jsx
<C
  propA="longlonglonglonglonglonglonglonglong"
  propB="longlonglonglonglonglonglonglonglong"
/>;

```

Reply:
Oh OK so it’s only inlined if it’s an opening element.